module/criteriaQuestions/helper.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * name : criteriaQuestions.js
+ * author : Aman
+ * created-date : 24-Jun-2020
+ * Description : Criteria questions related information.
+ */
+
+/**
+ * CriteriaQuestionsHelper
+ * @class
+*/
+
+module.exports = class CriteriaQuestionsHelper {
+
+     /**
+      * List of criteria questions
+      * @method
+      * @name list
+      * @param {Object} [ findQuery = "all" ] - filtered query data.
+      * @param {Array} [ fields = "all" ] - fields to include.
+      * @param {Array} [ skipFields = "none" ] - skip fields.   
+      * @returns {Array} List of criteria questions.
+     */
+
+    static list( findQuery = "all", fields = "all", skipFields = "none" ) {
+        return new Promise(async (resolve, reject) => {
+            try {
+        
+                let queryObject = {};
+
+                if( findQuery !== "all" ) {
+                    queryObject = findQuery;
+                }
+        
+                let projection = {};
+        
+                if (fields != "all") {
+                    fields.forEach(field => {
+                        projection[field] = 1;
+                    });
+                }
+
+                if (skipFields != "none") {
+                    skipFields.forEach(element => {
+                        projection[element] = 0;
+                    });
+                }
+        
+                let criteriaQuestionDocuments = 
+                await database.models.criteriaQuestions.find(
+                    queryObject, 
+                    projection
+                ).lean();
+                
+                return resolve(criteriaQuestionDocuments);
+                
+            } catch (error) {
+                return reject(error);
+            }
+        });
+    }
+
+    /**
+   * Create criteria Questions.
+   * @method
+   * @name create
+   * @param {String} criteriaId - criteria id.
+   * @param {Object} [ dataToUpdate = "" ] - Required data to update.
+   * @returns {JSON} - Create criteria question.
+   */
+
+    static create( criteriaId,dataToUpdate = "" ) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let criteriaModel = 
+                Object.keys(criteriaSchema.schema);
+                
+                let criteriaQuestionData = await this.list({
+                    _id : criteriaId
```

Comment:
@aman-tunerlabs - Put a filter of frameworkCriteriaExists or not here only, no need to put that logic everywhere